blog/_src/posts/2023-11-14-racket-v8-11.md

```diff
@@ -6,46 +6,34 @@
 
 Racket version 8.11 is now available from
 
-    https://racket-lang.org/
+    [https://download.racket-lang.org/](https://download.racket-lang.org)
```

Comment:
Because this line is indented, it is considered a codeblock, so you can't use the Markdown markup here.

Reply:
I would just combine this line and the previous line together.

Reply:
done

Reply:
do you resolve the conversation or do I? I'm never sure.

Reply:
You can (and did) resolve it.